src/Microsoft.AspNet.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -10,102 +10,102 @@
 
 namespace Microsoft.AspNet.Hosting
 {
-    public static class WebApplicationBuilderExtensions
+    public static class WebHostBuilderExtensions
```

Comment:
We make 2 files. One for extensions on `IWebHost`, one for extensions on `IWebHostBuilder`:

`WebHostBuilderExtensions`
`WebHostExtensions`

The services one can be called:

`WebHostServicesExtensions`